invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/BookDetails/BookDetails.scss

```diff
@@ -1,15 +1,17 @@
-.list {
-  overflow-wrap: break-word;
-}
+.ui.segment.book-metadata {
```

Comment:
As pointed out to @equadon, do not override the frameworks classes. 

You can wrap your root component and then override all you want inside it.
`
.book-metadata { 
  @extend ui.segement;
  h1 {...};
}`